keras_cv/layers/object_detection/roi_sampler.py

```diff
@@ -87,7 +87,7 @@ def call(
         Args:
           rois: [batch_size, num_rois, 4]
           gt_boxes: [batch_size, num_gt, 4]
-          gt_classes: [batch_size, num_gt, 1]
```

Comment:
I somehow remember that if it's without the last dimension, some weird broadcast will happen and the training result would be bad, just make sure you have run both OD models end to end before final merge

Reply:
reverted